Library/Homebrew/sandbox.rb

```diff
@@ -20,6 +20,13 @@ def self.print_autodisable_warning
     end
   end
 
+  def self.print_sandbox_message
+    unless @print_sandbox_message
```

Comment:
I find this variable name confusing, especially given it has the same name as the method. Maybe something like `printed_message`?